deepset_cloud_sdk/workflows/async_client/files.py

```diff
@@ -132,7 +132,7 @@ async def upload(
     """Upload a folder to deepset Cloud.
 
     :param paths: Path to the folder to upload. If the folder contains unsupported files, they're skipped.
-    during the upload. Supported file formats are TXT and PDF.
+    during the upload. Supported file formats are txt, pdf, docx, pptx, xlsx, xml, csv, html, md, json.
```

Comment:
to be honest, I'd drop it from the docstring. This is just gonna outdate. We can point to the `ALLOWED_TYPE_SUFFIXES` though?

Reply:
Thanks for the feedback. I doubted this as well. Will remove!